### ISSUE TITLE
base64: invalid input

### ISSUE THREAD

**Issue by MrAndrew** [w9jds/firebase-action#104]
Haven't changed anything with my service account key. Was working then all of a sudden the action today fails and prints this line: `base64: invalid input` 

I noticed a recent change to this repo's Dockerfile. Did these changes affect the key input format?

**Comment by MrAndrew** [comment]
Update, I found the issue. I was setting my key like this: `GCP_SA_KEY: ${{ secrets.MY_BASE_64_KEY }}}`  and when I removed the extra curly brace, it was fine. `GCP_SA_KEY: ${{ secrets.MY_BASE_64_KEY }}`  works as intended.

Thing is though, according to my Git history, this typo was being ignored for a few days and this action worked as intended. (successful deploys) Could the Dockerfile change have affected that? Like would an extra curly brace at the end of a base64 input be ignored before commits 6b70e5d/0cd4655 but be caught after?

**Comment by w9jds** [comment]
I don't think so, that is a system that GitHub reads and uses to pass arguments to the docker system so I would assume this is related to actions.

**Comment by MrAndrew** [comment]
Yes, but is it possible that previously this action was reading the expected size of the base64 encoded key which would cause a trailing } to be ignored? (I'm really hoping this is the case so I don't have to hunt down either a GitHub error or someone/thing messing with my version history.)
